python/pyspark/sql/functions.py

```diff
@@ -2341,6 +2415,7 @@ def approx_count_distinct(col: "ColumnOrName", rsd: Optional[float] = None) -> C
         return _invoke_function("approx_count_distinct", _to_java_column(col), rsd)
 
 
+@try_remote_functions
 def broadcast(df: DataFrame) -> DataFrame:
```

Comment:
ditto: PySpark DF -> Connect DF conversion

Reply:
Here too. The whole logic would look like:

```python
def broadcast(df: PySparkDataFrame) -> PySparkDataFrame:
    if is_remote():
        return connect.broadcast(df)
    else:
        return pyspark.boradcast(df)
```

And, we will just use the existing type annotations so users codes won't need to change their type annotations.